pkgs/development/python-modules/xmlrunner/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "xmlrunner";
+  version = "1.7.7";
```

Comment:
This package hasn't seen a release in 5 years.

Plus i think you get the same functionality if you do `pytest --junit-xml=path`

Reply:
Let me figure out if this package is actually needed in our environment, since I am generating Nix expressions for a large number of pip packages so that the Nix-based environment is equivalent (or better) than the existing one.

Reply:
This is a dependency of a python-irodsclient:
https://github.com/irods/python-irodsclient/blob/v0.8.2/setup.py

Which I would like to add in an upcoming PR from:
https://github.com/NixOS/nixpkgs/compare/master...WorldQuant:wq_python-irodsclient?expand=1#diff-b1d0e98bcf35fca90fb661d9052e4135